application/__tests__/example.macro.test.js

```diff
@@ -116,7 +123,14 @@ describe('Single page example macro english', () => {
     })
   })
 
-  test('Should include the escaped HTML markup from the examples', () => {
+  it('should have a button to copy Nunjucks code examples', () => {
+    const codeContainer = document.querySelector('#example-default-nunjucks')
+    const copyButton = codeContainer.querySelector('.app-link--copy')
+    expect(copyButton).not.toBeNull()
+    expect(copyButton.text).toBe('Copy')
```

Comment:
Can we test that the (correct) contents is copied too?

Reply:
We don't need to test the functionality of the third party component however we are testing it's instantiation by testing the 'onSuccess' method